server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const port = process.env.PORT || 3030;
+const baseRoute = require('./app/routes/base');
```

Comment:
Best to finish your requires before you start variable assignment. Possible refactor:

```
'use strict';

const express = require('express');
const bodyParser = require('body-parser');
const morgan = require('morgan');
const baseRoute = require('./app/routes/base');

const app = express();
const port = process.env.PORT || 3030;
```